lib/em-hiredis/connection_manager.rb

```diff
@@ -154,10 +154,6 @@ def on_disconnected(prev_state)
       # state when we emit :disconnected and :reconnected, so we should only
       # proceed here if our state has not been touched.
       return unless @sm.state == :disconnected
-
-      if @reconnect_attempt > 3
```

Comment:
I wonder if we could copy how the redis library does this and make this configurable, leaving 3 as the default.
That way we don't break anything that depends on the current behaviour

Reply:
Can do something similiar